components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -960,55 +969,184 @@
                         <DragIcon />
                       </button>
                     </td>
+                    <td
+                      class:flex={field.description}
+                      class={"title-and-description"}
+                    >
+                      <span>{field.title ?? "—"}</span>
+                      {#if field.description}
+                        <span class="description">{field.description}</span>
+                      {/if}
+                    </td>
+                    <td class="type">{field.type ?? "—"}</td>
+                    <td class="required">
+                      {#if field.required}
+                        <CheckmarkIcon />
+                      {/if}
+                    </td>
+                    <td>
+                      <button
+                        class="edit"
+                        on:click={() => editCustomField(field)}
+                      >
+                        <PencilIcon />
+                        <span>Edit</span>
+                      </button>
+                    </td>
                   </tr>
+                  {#if currentlyEditedField === field.id}
+                    <tr class="edit-custom-field">
+                      <td colspan="5">
+                        <div class="header">
+                          <h3>Edit field</h3>
+                          <button
+                            class="delete"
+                            on:click={() => removeCustomField(field)}
+                          >
+                            <TrashIcon />
+                            <span>Delete</span>
+                          </button>
+                        </div>
+                        <div class="input-field">
+                          <label>
+                            <div>Title</div>
```

Comment:
I recommend changing this to a span as it's typically used to style text within a label or other text containing element

Reply:
I left these as divs because I'd need to style spans with `display: block` to match the designs. Should they still be spans in this case?